lib/rubocop/ast/node/str_node.rb

```diff
@@ -13,6 +13,7 @@ class StrNode < Node
         :q => /\A%q/,
         :Q => /\A%Q/
       }.freeze
+      private_constant :PERCENT_LITERAL_TYPES
```

Comment:
When looking at `ArrayNode` I noticed I forgot to mark this constant as private for `StrNode` so I threw that in here too.